src/components/TxData.js

```diff
@@ -184,7 +186,8 @@ class TxData extends React.Component {
     ];
 
     try {
-      const walletAddressesMap = await this.props.wallet.checkAddressesMine(addresses);
+      const wallet = getGlobalWallet();
+      const walletAddressesMap = await wallet.checkAddressesMine(addresses);
```

Comment:
```suggestion
      const { checkAddressesMine } = getGlobalWallet();
      const walletAddressesMap = await checkAddressesMine(addresses);
```

Reply:
It's not possible to use this syntax with a HathorWallet method: it crashes the application.